src/NuProj.Targets/NuProj.props

```diff
@@ -16,6 +16,7 @@
     <EmbedSourceFiles Condition=" '$(EmbedSourceFiles)' == '' ">False</EmbedSourceFiles>
     <RequireLicenseAcceptance Condition=" '$(RequireLicenseAcceptance)' == '' ">False</RequireLicenseAcceptance>
     <DevelopmentDependency Condition=" '$(DevelopmentDependency)' == '' ">False</DevelopmentDependency>
+    <PublishPackage Condition=" '$(PublishPackage)' == '' ">Auto</PublishPackage>
```

Comment:
No, I don't think we can accept this. Publishing a package should *not* be done as part of a Build. No way. 
As part of another MSBuild target (maybe Deploy) *perhaps*. But absolutely not can we have Build, which is generally a local machine operation, publish the result somewhere.